projects/exui-common-lib/src/lib/gov-ui/components/gov-uk-date/gov-uk-date.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="govuk-form-group" [ngClass]="{'govuk-form-group--error': errorMessage?.isInvalid}" [formGroup]="formGroup">
+<div id="date-input-container" class="govuk-form-group" [ngClass]="{'govuk-form-group--error': errorMessage?.isInvalid}" [formGroup]="formGroup">
```

Comment:
@johnbenjamin-hmcts - what happens if there is more than one `gov-uk-date` component on a page... maybe a problem with duplicate IDs? How about something that's a bit more dynamic, based on the `config` that is passed in? For example, you could do: `<div [id]="config.id+'-date-input-container'">` or similar. I appreciate you'll have to change the CSS selector in `gov-uk-date.component.scss`.